incubator/nodejs-express/README.md

```diff
@@ -1,79 +1,65 @@
 # Node.js Express Stack
 
-This provides a prototype platform definition for Node.js Express based projects. This prototype uses Node.js v10 and Express v4.16.0
+The Node.js Express stack extends the [Node.js stack](../nodejs/README.md) and provides a consistent way of developing web applications using [Express](https://expressjs.com). Express is a fast, minimal and flexible Node.js web application framework that provides a robust set of features for web and mobile applications.
 
-## Stacks
+This stack is based on `Node.js v10` and `Express v4.16.0` and includes enables health checking and application metrics out of the box. The stack also provides a set of tests written using [Mocha](https://mochajs.org) test framework while the application metadata is provided via a `package.json` file.
```

Comment:
include enables?

Reply:
Adding the and a , helps:

The stack also provides a set of tests written using the [Mocha](https://mochajs.org) test framework, while the application metadata is provided via a `package.json` file.

Reply:
changed